docker/dhis.conf

```diff
@@ -0,0 +1,10 @@
+connection.dialect = org.hibernate.dialect.PostgreSQLDialect
+connection.driver_class = org.postgresql.Driver
+connection.url = jdbc:postgresql://db/dhis2
+connection.username = dhis
+connection.password = dhis
+
+tracker.import.preheat.cache.enabled=off
+
+flyway.migrate_out_of_order = true
+flyway.repair_before_migration = true
```

Comment:
Do we want this enabled by default?

Reply:
I suppose this is only for development environments so it should be ok...

Reply:
I don't know if we do, I can take it out. This is just sent around the backend channel every few weeks 😂 so I have to do it regularly. As its for development purposes only I think its ok. This will not be used in the e2e tests.

Reply:
I'll take it out as this can be added at a later point if needed 👌🏻

Reply:
Might be nice to add, as long as this won't leak into production environments 👍